### ISSUE TITLE
Inflammation of the cecum

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11579]
**Preferred term label:**
Inflammation of the cecum

**Proposed Definition**
Inflammation of the cecum is the presence of inflammatory cell infiltrates, crypt abscesses, and mucosal architectural distortion within the cecal mucosa.

**Parent term (use hpo.jax.org/app)**
Inflammation of the large intestine HP:0002037

**To be discussed**
According to https://pubmed.ncbi.nlm.nih.gov/31884010/ there are a few items that we should consider

1. In UC, **active inflammation** (neutrophils in the lamina propria and crypts) is distinguished from **non-active inflammation** (diffuse intramucosal inflammation, basal plasmacytosis, eosinophils). We may not want to have distinct terms, but we should add least be precise about what we mean by "inflammatory cells" (there are many different types). Also, I believe it is possible to have some degree of inflammation without (yet) having architectural distortion, and so is it good to stipulation this in the definition?

2. Should we state whether the inflammation needs to be transmucosal (which is characteristic of UC)?

3. Do we need a term for the related finding of **Basal plasmacytosis** (three or more plasma cells accumulate in the mucosal lamina propria between the crypt base and the muscularis mucosae)

4. The article has a description of mild, moderate and severe stages of inflammation related to UC. Should we include some of this in a Comment to "Inflammation of the large intestine HP:0002037"?

5. Histological vs endoscopic inflammation. The above definition appears to be for histology. It is probably important to state that "microscopic evidence of inflammation persists in 16%–100% of patients with endoscopically quiescent colitis" (in UC) and to be explicit about whether we mean biopsy, endoscopy or "either/or".

**Comment by pnrobinson** [comment]
The following are characteristics of CD
1. "Neutrophils are indicative of inflammation even when present exclusively in the stroma without attacking glands, causing cryptitis, or inducing microabscesses. The presence of deep transmural lymphoid aggregates, corresponding to areas of mucosal ulceration, is a particular morphological characteristic of CD that is often seen at disease onset. " => Do we need to say whether the inflammation is localized near specific cells types/stroma etc?
2. "Granulomas are localized aggregates of inflammatory cells including macrophages and T lymphocytes, and should be distinguished from aggregates of histiocytes found near glands or broken crypts."  => Should we include granulomas as a part of the inflammation terms or separately?
3. Will we also want to have Dysplasia terms?
"In low-grade dysplasia, crypt architecture shows minimal distortion and the cytological atypia is limited to the lower half of the crypts. Cytologically, the nuclei have slight hyperchromasia and the nuclear membrane has irregular edges. Dystrophic goblet cells (hypermucinous dysplasia) may be evident. High-grade dysplasia is characterized by greater architectural complexity. The cells show more atypia, with marked nuclear pleomorphism, an irregular nuclear membrane, and a macronucleolus with greater loss of polarity in the crypts and consequently greater nuclear stratification."

**Scoring systems**
I see that several distinct scoring systems are used. Should we try to ensure that our terms correspond to the items of these systems?
https://pmc.ncbi.nlm.nih.gov/articles/PMC10673803/


**General**
1. What about oedema and hyperaemia -- these are common features of inflammation? Should they be mentioned in the definition?
2. We need to say whether an HPO term is referring to a specific biopsy or to a summary of several biopsies. See "For a reliable diagnosis of IBD, at least two endoscopic biopsies each should be taken from at least five sites along the ileum, colon and rectum. If Crohn’s disease is suspected, endoscopic biopsy of the upper gastrointestinal (GI) tract is also suggested. "
